backend/core/src/seeder/seeds/applications.ts

```diff
@@ -226,15 +229,70 @@ export const makeNewApplication = async (
   app: INestApplicationContext,
   listing: Listing,
   unitTypes: UnitType[],
-  user?: User
+  user?: User,
+  pos = 0
 ) => {
-  const dto: ApplicationCreateDto = JSON.parse(JSON.stringify(applicationCreateDtoTemplate))
+  let dto: ApplicationCreateDto = JSON.parse(JSON.stringify(applicationCreateDtoTemplate))
   dto.listing = listing
   dto.preferredUnit = unitTypes
+  const splitEmail = dto.applicant.emailAddress.split("@")
+  dto.applicant.emailAddress = `${splitEmail[0]}${pos ?? ""}@${splitEmail[1]}`
+  dto.applicant.firstName = `${dto.applicant.firstName}${pos ?? ""}`
+  dto.applicant.lastName = `${dto.applicant.lastName}${pos ?? ""}`
   const applicationRepo = app.get<Repository<Application>>(getRepositoryToken(Application))
-  return await applicationRepo.save({
+  const originalApp = await applicationRepo.save({
     ...dto,
     user,
     confirmationCode: ApplicationsService.generateConfirmationCode(),
   })
+
+  if (pos === 0) {
```

Comment:
I thought we were changing all of the duplicate finding logic and that that work was TBD? Just want to make sure we aren't doing work we won't need in general for the backend.

Reply:
I think this should be ok, since it's manually saving to the afs repo. The new logic, with the first iteration anyway, mostly just moves the logic to a cron job with some performance improvements. We will have to make an update, since it adds a new key, but it should be a quick update in either PR, depending on which one gets merged first.